state/azure/tablestorage/tablestorage.go

```diff
@@ -254,30 +252,26 @@ func getTablesMetadata(metadata map[string]string) (*tablesMetadata, error) {
 	return &meta, nil
 }
 
-func (r *StateStore) writeRow(req *state.SetRequest) error {
+func (r *StateStore) writeRow(ctx context.Context, req *state.SetRequest) error {
 	marshalledEntity, err := r.marshal(req)
 	if err != nil {
 		return err
 	}
 
-	writeContext, cancel := context.WithTimeout(context.Background(), timeout)
```

Comment:
Please don't remove the background. Still use a context from `ctx` with the timeout

Reply:
Fixed.